static/js/apps/biomed_nl/app.tsx

```diff
@@ -21,24 +21,51 @@
 
 import { css, ThemeProvider } from "@emotion/react";
 import axios from "axios";
-import _ from "lodash";
 import queryString from "query-string";
 import React, { ReactElement, useEffect, useState } from "react";
 import Collapsible from "react-collapsible";
 import ReactMarkdown from "react-markdown";
-import { Input } from "reactstrap";
+import { Input, InputGroup } from "reactstrap";
 import rehypeRaw from "rehype-raw";
 import remarkGfm from "remark-gfm";
 
+import SimpleText from "../../components/content/simple_text";
+import { Search } from "../../components/elements/icons/search";
+import { Link, LinkBox } from "../../components/elements/link_box";
 import theme from "../../theme/theme";
 import { stringifyFn } from "../../utils/axios";
 import { updateHash } from "../../utils/url_utils";
+import { SpinnerWithText } from "./spinner";
 
 // Constants for URL hash parameters
 const URL_HASH_PARAMS = {
   q: "q",
 };
 
+const sampleQuestions = [
```

Comment:
nit: global constants should be named using UPPER_SNAKE_CASE

Reply:
ah sorry, thanks for pointing this out -- fixed!